src/main/resources/fabric.mod.json

```
@@ -26,9 +26,9 @@
     "dsurround.mixins.json"
   ],
   "depends": {
-    "fabricloader": ">=0.11.7",
+    "fabricloader": ">=0.12.6",
```

The minimum loader version should be the one introduced to patch the Log4Shell exploit, imo.
```suggestion
    "fabricloader": ">=0.12.11",
```